### DISCUSSION TITLE
Nested drawer is nice, but sometimes I'd like to open in a new tab, especially when nested drawer has bugs.

### DISCUSSION THREAD

**Discussion by Stupidism** [payloadcms/payload#2958] [Feature Requests & Ideas]
1. https://github.com/payloadcms/payload/issues/2815#issuecomment-1612715760
2. https://github.com/payloadcms/payload/issues/2945
3. https://github.com/payloadcms/payload/issues/2945#issuecomment-1617384822

My suggestion:

Allow customize an option to open nested documents in new tabs.
Or toggle this feature when pressing `Command` key like open the links above.

**Comment by DanRibbens** [comment]
Thanks for discussion @Stupidism,

We certainly won't give up on fixing these drawer issues. That has the biggest impact for everyone and serves the greatest needs first.

The drawer is widely used pattern in the UI, for this discussion are you talking about the relationship and upload field inputs?

Similar to what you've identified, I'd like to see additional Relationship field UIs that can be chosen from that would maybe replace your UI concerns in a different way. For example, embedding a collection list table directly in the form for managing `hasMany` relations and there are probably other different ways to present the input. It may be that your concepts are managed through the same properties or that it could solve a similar use-case for extending the admin UI so that opening in a new tab isn't even necessary.

What kind of situations would you reach for an open-in new tab option version instead of having the drawer open or would you rather have some new field relationship UI that solves the need better?

**Reply by Stupidism** [reply]
For example when I want to edit every one in the list. I'd like to open 5 tabs at once and close tabs one by one after editing them.
It's more natural for me to make sure I've edited them all.

With drawer way, I need to open one drawer, edit it and close drawer, I might forget where am I when I choose the next relationship chip.

**Reply by Stupidism** [reply]
Maybe at least let me open a new tab when I already open a drawer but found it's buggy or hard to edit?

<img width="1257" alt="image" src="https://github.com/payloadcms/payload/assets/5801015/da9841fa-fa9d-4562-9e12-1f821ab66a3d">

The original problem was hard to edit related documents. 

Opening in drawer is a nice solution to this problem, it keeps you focused on the original document.
But opening in new tab is also a solution when you just want to navigate to the related document instead of focusing on the original one.
